.circleci/config.yml

```diff
@@ -81,8 +81,15 @@ jobs:
             npm install
             npm run test.saucelabs.us
       - run:
-          name: "Run Nightwatch"
+          name: "Testrunner Toolkit Cypress examples"
```

Comment:
This fails, do we already want to run this in CI?

Reply:
@wswebcreation thanks for being on top of this! I'm working on fixing it right now 👍

Reply:
Please remove SauceCTL from the pipeline, we haven't implemented it yet here in this CI (not as far as I know)